sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/MinedOpinion.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.util.IterableStream;
+
+/**
+ * The {@link MinedOpinion} model.
+ */
+public final class MinedOpinion {
+    private final AspectSentiment aspect;
+    private final IterableStream<OpinionSentiment> opinions;
+
+    /**
+     * Create an {@link MinedOpinion} model that describes mined opinion.
+     *
+     * @param aspect The aspect of the mined opinion.
```

Comment:
nit: I think [this](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py#L793) might be a better description